styles/widgets/material/toolbar.material.less

```diff
@@ -192,7 +192,6 @@
     .dx-button-has-text:not(.dx-button-back) {
         .dx-icon {
             .dx-icon-sizing(@MATERIAL_BUTTON_ICON_SIZE);
-            .dx-icon-margin(6px);
```

Comment:
Is it changed the appearance of other button types in dxToolbar?

Reply:
fixed it